src/main/java/io/github/railroad/project/ui/welcome/WelcomeProjectsPane.java

```diff
@@ -12,17 +13,28 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static io.github.railroad.Railroad.manager;
+
 public class WelcomeProjectsPane extends ScrollPane {
     private final ListView<Project> projectsList = new ListView<>();
+
     private ObservableValue<ProjectSort> sortProperty;
 
-    public WelcomeProjectsPane() {
+    public WelcomeProjectsPane(ProjectSearchField searchField) {
+        manager.loadProjects();
         setStyle("-fx-background-color: #f0f0f0;");
         setFitToWidth(true);
         setFitToHeight(true);
         setHbarPolicy(ScrollBarPolicy.NEVER);
         setVbarPolicy(ScrollBarPolicy.AS_NEEDED);
 
+        searchField.textProperty().addListener(obs->{
+            String filter = searchField.getText();
+            filterProjects(filter);
+
+        });
+
+
```

Comment:
```suggestion
        searchField.textProperty().addListener(obs->{
            String filter = searchField.getText();
            filterProjects(filter);
        });
```
Just some unnecessary whitespace.